core/src/test/scala/pickling/neg/java-field-fail.scala

```diff
@@ -0,0 +1,20 @@
+package scala.pickling.javafieldfail
+
+import scala.pickling._
+import NegativeCompilation._
+import org.scalatest.FunSuite
+
+class JavaFieldFailTest extends FunSuite {
+  test("main") {
+    expectError("Cannot generate") {
```

Comment:
REPL gives me what we want, 

```
scala> Pickler.generate[FakeByte]
<console>:18: error: exception during macro expansion:
scala.pickling.PicklingException: No fields are captured. You need a custom pickler to handle this.
	at scala.pickling.PicklerMacros$$anonfun$pickleLogic$1.unifiedPickle$1(Macros.scala:240)
```

but I get an error that I don't want using the negative test:

```
[info] macro implementation not found: generate
[info] (the most common reason for that is that you cannot use macro implementations in the same compilation run that defines them)
```

so I am guessing it's specific to Toolbox used by negative test.